src/librustdoc/html/static/rustdoc.css

```
@@ -126,6 +126,10 @@ h1, h2, h3, h4,
 	font-family: "Fira Sans", sans-serif;
 }
 
+.content ul.crate a.crate {
+	font: 16px/1.6 "Fira Sans";
```

Why doesn't this have a fallback font?

```suggestion
	font: 16px/1.6 "Fira Sans", sans-serif;
```